src/main/resources/application.conf

```
@@ -44,6 +44,12 @@ party-registry-proxy {
       institutions {
         open-data-url = ${PARTY_REGISTRY_INSTITUTIONS_URL}
       }
+      AOO {
+        open-data-url = ${PARTY_REGISTRY_AOO_URL}
+      }
+      UO {
```

very minor naming convention
```suggestion
      aoo {
        open-data-url = ${PARTY_REGISTRY_AOO_URL}
      }
      uo {
```